fe/src/component/bottom/Slides.js

```diff
@@ -1,41 +1,43 @@
 import React, { useState, useEffect } from "react";
 import Slide from "./Slide";
 import More from "./More";
+import constansts from "../../constants/constansts";
+import { myfetch } from "../../utils/utils";
 
 const Slides = () => {
   const [cardsInfor, setCardInfor] = useState([]);
   const [isDisplayed, setIsDisplayed] = useState(false);
-  const myfetch = (url) => {
-    return fetch(url).then((res) => res.json());
-  };
 
   useEffect(() => {
-    const url = [
-      "https://api.codesquad.kr/onban/side",
-      "https://api.codesquad.kr/onban/soup",
-      "https://api.codesquad.kr/onban/main",
-    ];
-
-    const data = Promise.all(
-      url.map(async (v) => {
-        return await myfetch(v);
-      })
+    const url = constansts.SlideINfor.map(
+      (v) => `${constansts.MAIN_API_URL}${v.postfix}`
     );
 
-    data.then((res) => setCardInfor(res));
+    const data = Promise.all(url.map(myfetch));
+    data
+      .then((res) =>
+        constansts.SlideINfor.map((v, i) =>
+          Object.assign(v, {
+            data: res[i].data,
+          })
```

Comment:
```suggestion
        constansts.SlideINfor.map((v, i) => (
          {...v, data: res[i].data}
          )
```
객체를 스프레드 시켜서 매핑시켜줄 수도 있겠네요! 

<img width="226" alt="Screen Shot 2022-05-01 at 12 47 43" src="https://user-images.githubusercontent.com/30427711/166131384-ea1a0bfb-d5b6-4bd2-a658-d44756180deb.png">

그리고 Object.assign은 첫번째 인자로 들어온 원본 객체를 변경시키기 때문에 차이를 알고 계시면 좋을 것 같습니다!